### DISCUSSION TITLE
Why there are 2 models, User and Profile?

### DISCUSSION THREAD

**Discussion by yardenj2r** [prisma/prisma#17454] [Q&A] [answered]
Hi there! How are you doing?

I'm reading about the following template: https://vercel.com/templates/next.js/fullstack-prisma-nextjs-blog

Why there are 2 models, User and Profile? Why not only User or only Profile? I couldn't understand. Can someone explain, please?

Thank you!

**Comment by nurul3101** [comment, marked as answer]
Hey @yardenj2r 👋 

Can you please elaborate, I didn't quite get the statement

> Why there are 2 models, User and Profile? Why not only User or only Profile? 

Do you intend to say that only one model is required? In that case, this is just an example for modelling, if in your use case all the profile fields could be accommodated in user models, then you can just have a single user model and that should also work.

**Reply by yardenj2r** [reply]
Hi @nurul3101 ! How are you doing? First of all, thank you for your answer. What I mean is why would someone want to separate these into 2 different models? Or at least at this case, why did they do it like that?Is this way the best to be used with Prisma for some reason?

**Reply by nurul3101** [reply]
In this case, it doesn't make sense to have a separate profile model to store the bio, but the main purpose is to show a one-to-one relation between the user and profile models.

**Reply by yardenj2r** [reply]
I see, thank you very much!

**Reply by nurul3101** [reply]
No worries! 😄

**Comment by Uncle798** [comment]
I think in this case the User model stores information that the end user has given the example app. The Account model is used to store information retrieved from the authentication provider. The relationships then tie the two together.

**Reply by yardenj2r** [reply]
And is there a good reason to separate like that? Would you do it in a real product? Thanks for your reply!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
